spec/kissable/ab_spec.rb

```diff
@@ -5,6 +5,7 @@
   let(:groups) { nil }
   let(:ratios) { nil }
   let(:cookies) { {} }
+  let(:login) { "jsmith@example.com" }
```

Comment:
Trivial, but, you should use single quotes over double quotes if no interpolation is needed.